l_russian:
 travel_danger_events.3000.t: "$travel_danger_prefix$ на горной тропе"
 travel_danger_events.3000.desc.first_time: "Аккуратно, шаг за шагом, я медленно двигаюсь вперед по узкой горной тропе. Вдруг рядом раздается грохот, а за ним — короткий вскрик.\n\nПерепугавшись, я резко оборачиваюсь и вижу, что [fallen_victim.GetFirstName] за мной больше не идет. Вместо этого на меня бросает столь же изумленный взгляд [char_next_in_line.GetFirstName]. Перед нами только камни, вырванные из земли какой-то силой.\n\n«Где [fallen_victim.GetFirstNameNoTooltip]?» — спрашиваю я требовательным тоном. Слова едва срываются с пересохшего языка.\n\nВ ответ лишь тихо шумит ветер."
 travel_danger_events.3000.desc.second_time: "Я долго иду по узкой и, кажется, бесконечной горной тропе. Усталость постепенно приглушает все мысли. Ветер начинает обманчиво шептать на ухо полузабытыми голосами.\n\nИз этого состояния меня вскоре вырывает вполне настоящий крик. По спине пробегает холодок. Медленно развернувшись, я вновь вижу, что нас стало на одного меньше. [fallen_victim.GetFirstName] — очередной выпавший из наших рядов.\n\n«Неужели мы… потеряли… еще одного?» — медленно произношу я. [char_next_in_line.GetFirstName] не отвечает, но по [char_next_in_line.GetHersHis] окаменевшему от ужаса лицу я и так всё понимаю."
 travel_danger_events.3000.a.first_time: "Смотрите под ноги! Идем дальше."
 travel_danger_events.3000.a.second_time: "Эх, [fallen_victim.GetFirstNameNoTooltip]! Тебе же #EMP говорили#!, смотри под ноги!"
 travel_danger_events.3000.a.tt: "Эти коварные горы забрали еще одну жизнь — не первую и далеко не последнюю."
 travel_danger_events.3000.b: "[fallen_victim.GetFirstNameNoTooltip]! [fallen_victim.GetFirstNameNoTooltip]! Мы спасем тебя!"
 travel_danger_events.3000.b.tt: "Вы и ваша свита попытаетесь провести спасательные работы. #EMP #X Это очень опасно.#! #!Многие погибнут."
 travel_danger_events.3000.b.found_nothing: "Найти персонажа [fallen_victim.GetFirstNameNoTooltip] не удалось."
 travel_danger_events.3000.b.found_cloth: "Удалось найти только окровавленный лоскут."
 travel_danger_events.3000.b.found_fallen_victim: "Как это ни странно, но [fallen_victim.GetFirstNameNoTooltip] остается в живых."
 travel_danger_events.3000.c.first_time: "Мы будем молиться за н[fallen_victim.GetHersHis]."
 travel_danger_events.3000.c.second_time: "[ROOT.Char.GetFaith.HighGodName], избави нас от опасности!"
 travel_danger_events.3000.d.first_time: "Даже мне бы так не справиться!"
 travel_danger_events.3000.d.second_time: "Во второй раз даже забавнее!"
 travel_danger_events.3001.t: "$travel_danger_prefix$ лавина!"
 travel_danger_events.3001.desc: "После долгого и опасного путешествия среди горных вершин я неторопливо вытягиваю руки и ноги к ревущему пламени очага местной таверны, а [first_avalanche_victim.GetFirstName] и [second_avalanche_victim.GetFirstName] занимаются снаружи нашими пожитками. Внезапно раздается далекий, но могучий, глубокий и вибрирующий рокот.\n\n«Лавина!» — кричит кто-то. Прежде чем кто-то успевает среагировать, звук вырастает до оглушительного рева, стены и пол сотрясаются. Мгновение длится целую вечность, а потом наступает еще более ужасная тишина."
 travel_danger_events.3001.a: "[ROOT.Char.GetCulture.GetName] умеют выживать в горах."
 travel_danger_events.3001.a.alternate: "[travel_leader.GetCulture.GetName] умеют выживать в горах."
 travel_danger_events.3001.a.flavor: "Это вы еще [ROOT.Char.GetCulture.GetCollectiveNoun|l]их лавин не видели!"
 travel_danger_events.3001.a.flavor.alternate: "[Select_CString(travel_leader.IsFemale, 'Моя', 'Мой' )] $travel_leader_court_position$ [travel_leader.GetFirstName] принял все необходимые меры предосторожности против опасностей, поджидающих в горах, — сыграло роль [travel_leader.GetHerHis] [travel_leader.GetCulture.GetCollectiveNoun|l]ое происхождение."
 travel_danger_events.3001.b: "Ждите, помощь уже в пути!"
 travel_danger_events.3001.b.flavor: "Когда под снегом гибнут люди, нужно спешить изо всех сил!"
 travel_danger_events.3001.b.success: "Быстрое спасение из-под лавины"
 travel_danger_events.3001.b.failure: "Затрудненное спасение из-под лавины"
 travel_danger_events.3001.c: "Я побуду внутри... какое-то время."
 travel_danger_events.3002.opening: "[ROOT.Char.GetTitledFirstName],"
 travel_danger_events.3002.desc: "С сожалением сообщаю, мой сюзерен, что владение [avalanche_location.GetName] недавно накрыло лавиной.\n\nОна унесла много жизней, а тех, кому посчастливилось уцелеть, включая проходивших теми краями путешественников, постигли серьезные лишения."
 travel_danger_events.3002.a.special: "Я знаю! Я как раз #EMP оттуда!#!"
 travel_danger_events.3002.a.default: "Ужас какой!"
 travel_danger_events.3002.b: "Это поможет им восстановиться."
 travel_danger_events.3003.t: "$travel_danger_prefix$ снегопад"
 travel_danger_events.3003.desc: "Снег валит уже несколько дней. Дорога впереди кажется совершенно непроходимой. Но вот я вижу впереди небольшую группу людей, которые стремительно пробираются к нам через сугробы. Вскоре ко мне подходит [maron_leader.GetWomanMan] и с широкой ухмылкой заявляет:\n\n«Похоже, вы попали в затруднительное положение. Но не волнуйтесь, за хорошую плату мы поможем вам продолжить путешествие»."
 travel_danger_events.3003.a: "Помощь ближнему — сама по себе награда!"
 travel_danger_events.3003.a.success: "Моя свита быстро начинает двигаться!"
 travel_danger_events.3003.a.failure: "Моя свита всё еще барахтается в снегу"
 travel_danger_events.3003.b: "Цена приемлемая."
 travel_danger_events.3003.c: "Мы сами справимся."
 travel_danger_events.3003.c.flavor: "Опыт зимних путешествий поможет нам выбраться... если он у нас есть."
 travel_danger_events.3003.c.complete_success: "Моя свита быстро начинает двигаться!"
 travel_danger_events.3003.c.partial_success: "Моя свита медленно начинает двигаться"
 travel_danger_events.3003.c.failure: "Моя свита барахтается в снегу"
 travel_danger_events.3004.t: "$travel_danger_prefix$ снежный шквал"
 travel_danger_events.3004.desc: "«Снежный шквал! Скорее в порт!»\n\nВ абсолютном спокойствии природы жуткий вопль рулевого звучит как-то странно. Но я уже слышу нарастающий рев стихии. Мы не успеваем и моргнуть, корабль погружается в мокрую белую мглу. С треском, заглушающим ветер, главная мачта ломается и падает за борт.\n\nТак же быстро, как появился, снежный шквал уходит прочь. Наше судно лениво покачивается на совершенно спокойной воде, волоча за собой обрывки парусов и обломки весел."
 travel_danger_events.3004.a: "Нужно как-то смастерить паруса!"
 travel_danger_events.3004.a.success: "Корабль снова готов плыть!"
 travel_danger_events.3004.a.failure: "Мы медленно дрейфуем по ветру"
 travel_danger_events.3004.b: "Не бойтесь! Паруса еще есть."
 travel_danger_events.3004.b.flavor: "Благодаря мореходному опыту моей свиты мы готовы к такому развитию событий. Новую снасть быстро мастерят из запасных парусов и деревянных балок."
 travel_danger_events.3004.c: "Нас #EMP кто-нибудь#! найдет... Надеюсь"
 travel_danger_events.3004.c.success: "Нам предлагают помощь с проходящего корабля"
 travel_danger_events.3004.c.failure: "Много дней в пределах видимости не показывается ни одно судно"
 travel_danger_events.3005.t: "$travel_danger_prefix$ пламя в ночи"
 travel_danger_events.3005.desc: "На исходе дня мы идем среди холмов. Налетевшая туча укрывает нас плотным пологом мрака. Вскоре издалека доносится громкий раскат грома. За ним следует другой, сопровождаемый ошеломляющей вспышкой молнии.\n\nСледующий разряд ярко освещает местность, и его след отпечатывается у меня на глазах. Проморгавшись, я вижу, куда ударила молния: в кустарник, который уже пылает ярким пламенем. И оно движется к нам!"
 travel_danger_events.3005.a: "БЕЖИМ!"
 travel_danger_events.3005.a.success: "Мы убегаем от пожара"
 travel_danger_events.3005.a.failure: "Пламя догоняет нас!"
 travel_danger_events.3005.b: "Погасим пламя, пока оно не распространилось!"
 travel_danger_events.3005.b.success: "Вам удается потушить пожар"
 travel_danger_events.3005.b.failure: "Пожар удалось потушить, но какой ценой?"
 travel_danger_events.3005.b.c_failure: "Мою свиту охватывает пламя!"
